packages/esm-patient-medications-app/src/add-drug-order/drug-search/drug-search.component.tsx

```diff
@@ -14,7 +15,8 @@ export default function DrugSearch({ openOrderForm }: DrugSearchProps) {
   const { t } = useTranslation();
   const isTablet = useLayoutType() === 'tablet';
   const [searchTerm, setSearchTerm] = useState('');
-  const debouncedSearchTerm = useDebounce(searchTerm);
+  const { debounceDelayInMs } = useConfig<ConfigObject>();
+  const debouncedSearchTerm = useDebounce(searchTerm, debounceDelayInMs ? debounceDelayInMs : 300);
```

Comment:
```suggestion
  const debouncedSearchTerm = useDebounce(searchTerm, debounceDelayInMs ?? 300);
```

Reply:
@ibacher thanks for the suggestion. I have fixed the issue